operations/iuf/stages/deliver_product.md

```diff
@@ -40,6 +40,20 @@ Artifacts will be uploaded to the following locations based on entries specified
 | VCS Content                   | `vcs`                             |
 | IMS Images and Recipes        | `ims`                             |
 
+The following table describes upload behavior when the artifact being uploaded already exists on the system.
+
+| Artifact                      | Upload Behavior                                          |
+| ----------------------------- | ---------------------------------                        |
+| S3 Content                    | overwrite existing artifact                              |
+| S3 Loftsman Manifests         | overwrite existing artifact                              |
+| Nexus Docker Registries       | new artifact uploaded with same tag as previous artifact |
```

Comment:
This behavior seems different than the past.   Are you sure this is what it does?   Also `Nexus Docker Registries` seems incorrect it should probably say `Nexus Docker Images`.   There's only one registry on the system.

Reply:
Yes, that should say ```Docker images``` - good catch!  The explanation came from a discussion at https://cray.slack.com/archives/C04593DNN3U/p1675365684982119 and we can resume it if you don't think it is accurate.